src/MgvLib.sol

```diff
@@ -53,30 +56,32 @@ library MgvLib {
    Some miscellaneous data types useful to `Mangrove` and external contracts */
   //+clear+
 
-  /* `SingleOrder` holds data about an order-offer match in a struct. Used by `marketOrder` (and some of its nested functions) to avoid stack too deep errors. */
+  /* `SingleOrder` holds data about an order-offer match in a struct. Used by `marketOrder` (and some of its nested functions) to avoid stack too deep errors. It is used in market order and cleaning. */
   struct SingleOrder {
     OLKey olKey;
     uint offerId;
+    /* The `offer` given to the maker will be cleaned of `prev`/`next` pointers. */
     Offer offer;
     /* `wants`/`gives` mutate over execution. Initially the `wants`/`gives` from the taker's pov, then actual `wants`/`gives` adjusted by offer's ratio and volume. */
     uint takerWants;
     uint takerGives;
     /* `offerDetail` is only populated when necessary. */
     OfferDetail offerDetail;
     Global global;
+    /* The `local` given to the maker will be cleaned of tick tree information. */
     Local local;
   }
 
   /* <a id="MgvLib/OrderResult"></a> `OrderResult` holds additional data for the maker and is given to them _after_ they fulfilled an offer. It gives them their own returned data from the previous call, and an `mgvData` specifying whether Mangrove encountered an error. */
 
   struct OrderResult {
-    /* `makerdata` holds a message that was either returned by the maker or passed as revert message at the end of the trade execution*/
+    /* `makerData` holds a message that was either returned by the maker or passed as revert message at the end of the trade execution*/
     bytes32 makerData;
     /* `mgvData` is an [internal Mangrove status code](#MgvOfferTaking/statusCodes) code. */
     bytes32 mgvData;
   }
 
-  /* `CleanTarget` holds data about an offer that should be cleaned, i.e. made to fail by executing it with the specified volume. */
+  /* `CleanTarget` holds data about an offer that should be cleaned, i.e. made to fail by executing it with the specified volume. It is used in `MgvOfferTaking.cleanByImpersonation`. */
   struct CleanTarget {
     uint offerId;
     Tick tick;
```

Comment:
"The events emitted for use by bots are listed here:" -> "The events emitted are listed here:"